src/resmom_win/start_exec.c

```diff
@@ -3465,7 +3467,8 @@ open_std_file(job *pjob, enum job_file which, int mode, gid_t exgid)
 
 		if (became_admin) {   /* go back to being user */
 			if (pjob->ji_user->pw_userlogin != INVALID_HANDLE_VALUE) {
-				(void)impersonate_user(pjob->ji_user->pw_userlogin);
+				if (impersonate_user(pjob->ji_user->pw_userlogin) == FALSE)
+					log_joberr(-1, __func__, "impersonate_user() failed", pjob->ji_qs.ji_jobid);
```

Comment:
Don't need the user (pw_name) to be printed here?

Reply:
fixed in https://github.com/openpbs/openpbs/pull/1794/commits/90ed7603b8b003d419447d749bc57f0e30543573